{"level":"error","message":"Erreur lors du traitement de la requête de connexion : The \"salt\" argument must be of type string or an instance of ArrayBuffer, Buffer, TypedArray, or DataView. Received undefined","timestamp":"2025-01-10T13:36:25.984Z"}
{"level":"error","message":"Erreur lors du traitement de la requête de connexion : The \"salt\" argument must be of type string or an instance of ArrayBuffer, Buffer, TypedArray, or DataView. Received undefined","timestamp":"2025-01-10T13:36:55.466Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:20:12.329Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:20:35.507Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:20:44.494Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:21:19.058Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:21:30.871Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:25:34.398Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:26:54.127Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:49:49.744Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:50:15.819Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:52:28.981Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:52:44.271Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:59:33.607Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:59:42.054Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:59:42.798Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T12:59:45.390Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T13:00:24.397Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T14:25:41.253Z"}
{"level":"error","message":"Erreur lors de la récupération du profil utilisateur : (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function","stack":"TypeError: (0 , _services_user_getUserProfileService__WEBPACK_IMPORTED_MODULE_1__.getUserProfileService) is not a function\n    at getUserProfileHandler (webpack-internal:///(api)/./src/handlers/user/getUserProfileHandler.ts:25:126)\n    at handler (webpack-internal:///(api)/./src/pages/api/user/profile.ts:8:103)\n    at K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2877)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)\n    at async DevServer.runApi (/app/node_modules/next/dist/server/next-server.js:598:9)\n    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)\n    at async DevServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:817:17)\n    at async /app/node_modules/next/dist/server/dev/next-dev-server.js:339:20\n    at async Span.traceAsyncFn (/app/node_modules/next/dist/trace/trace.js:154:20)\n    at async DevServer.handleRequest (/app/node_modules/next/dist/server/dev/next-dev-server.js:336:24)\n    at async invokeRender (/app/node_modules/next/dist/server/lib/router-server.js:173:21)\n    at async handleRequest (/app/node_modules/next/dist/server/lib/router-server.js:350:24)\n    at async requestHandlerImpl (/app/node_modules/next/dist/server/lib/router-server.js:374:13)\n    at async Server.requestListener (/app/node_modules/next/dist/server/lib/start-server.js:141:13)","timestamp":"2025-01-13T14:28:53.541Z"}
{"level":"error","message":"Erreur lors du traitement de la connexion : \nInvalid `prisma.user.findUnique()` invocation:\n\n\nCan't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.","timestamp":"2025-01-21T15:14:06.414Z"}
{"level":"error","message":"Erreur lors du traitement de la connexion : \nInvalid `prisma.user.findUnique()` invocation:\n\n\nCan't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.","timestamp":"2025-01-21T15:14:37.588Z"}
{"level":"error","message":"Erreur lors du traitement de la connexion : \nInvalid `prisma.user.findUnique()` invocation:\n\n\nerror: Error validating datasource `db`: You must provide a nonempty URL. The environment variable `DATABASE_URL` resolved to an empty string.\n  -->  schema.prisma:9\n   | \n 8 |   provider = \"postgresql\"\n 9 |   url      = env(\"DATABASE_URL\")\n   | \n\nValidation Error Count: 1","timestamp":"2025-01-22T10:07:42.793Z"}
{"level":"error","message":"Erreur lors du traitement de la connexion : \nInvalid `prisma.user.findUnique()` invocation:\n\n\nerror: Error validating datasource `db`: You must provide a nonempty URL. The environment variable `DATABASE_URL` resolved to an empty string.\n  -->  schema.prisma:9\n   | \n 8 |   provider = \"postgresql\"\n 9 |   url      = env(\"DATABASE_URL\")\n   | \n\nValidation Error Count: 1","timestamp":"2025-01-22T10:07:47.462Z"}
